src/formatDate.js

```diff
@@ -50,7 +50,62 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dayParts = date.split(fromFormat[fromFormat.length - 1]);
+
+  let day = '';
+  let month = '';
+  let year = '';
+  const devider = toFormat[toFormat.length - 1];
+
+  for (let i = 0; i < dayParts.length; i++) {
+    if (fromFormat[i] === 'DD') {
```

Comment:
Avoid using "magic values" (`DD`), create constants for this strings (`MM`, `YYYY`, `YY`) and reuse them.